go/vt/worker/restartable_result_reader.go

```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/golang/glog"
```

Comment:
This should instead be `vitess.io/vitess/go/vt/log`. It allows us to remap the logging functions as needed.